### ISSUE TITLE
Cam

### ISSUE THREAD

**Issue by openbrian** [giorgiosironi/phpunit-selenium#355]

**Comment by robgudgeon** [comment]
Agreed. This is an issue with quite a number of vendor SDKs that I use, especially since php-5.6

**Comment by giorgiosironi** [comment]
The way I would solve this in general is to use a HTTP request library, but here we are trying to keep dependencies to a minimum so the proposed solution.
I will merge a Pull Request adding this support, the test `Tests_Selenium2TestCase_Coverage_RemoteCoverageTest` should be extended to deal with this feature.

**Comment by openbrian** [comment]
Would you prefer a implicit stream context (http://php.net/manual/en/function.stream-context-set-default.php) or an explicit one like this file_get_contents($url, false, $streamContext) ?

**Comment by giorgiosironi** [comment]
The first one implies global state so the second is better imho.

**Comment by openbrian** [comment]
See #360.
